app/views/layouts/angular/_generic_form_buttons.html.haml

```diff
@@ -0,0 +1,27 @@
+.clearfix
```

Comment:
The new partial is devoid of `buttonGroupController`.
TBH, `buttonGroupController` isn't buying us anything... in fact, it's an overhead because it forces us to convert `buttonGroupController` to `controllerAs` also, which would not work with the other `non controllerAs` controllers.
So controllers that get converted to `controller as vm` should use this new partial going forward.

I'm open to other ideas if you have any.